composer.json

```diff
@@ -7,6 +7,7 @@
     }
   },
   "require": {
+    "php": ">=5.6.0",
```

Comment:
To my knowledge there is still a lot of CentOS 6 deployed with A2Billing and it would be to harsh to leave them hanging without the possibility to upgrade.
I see there is pleasant features in PHP 5.4 and above but until we really need them in a2billing code base, I will upvote to keep 5.3 as minimal PHP requirement.